lib/src/components/NxTabs/NxTabs.scss

```diff
@@ -71,14 +71,14 @@
     @include nx-text-helpers.semi-bold;
     color: var(--nx-color-text-dark);
```

Comment:
> In light mode, the text color changes, but it doesn't seem so in dark mode. (Also now nx-color-text-stark 😄 )

I wonder if that's a bug. I'll confirm with Jason about it.

Reply:
> > In light mode, the text color changes, but it doesn't seem so in dark mode. (Also now nx-color-text-stark 😄 )
> 
> I wonder if that's a bug. I'll confirm with Jason about it.

Looks like this is intentional. As per Jason, "Just checked, and yep, it's intentional. The change is necessary in Light Mode, for contrast reasons, but the contrast relationships are different enough in DM that its not necessary to lighten the text, and it would actually pop way too much if we did."

Reply:
Sounds good, just update the variable to stark 👍